network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/blog_card.html

```diff
@@ -12,7 +12,7 @@
 each blog page's dominant tag)
 
 {% block tags %}
-  {% include "./blog_card_topics.html" with topics=page.specific.topics.all %}
+  {% include "./blog_topics.html" with topics=page.specific.topics.all %}
```

Comment:
**kudos** thanks for the rename! This makes more sense to me and we can share this among other templates